### ISSUE TITLE
Notices are glitching while being dismissed

### ISSUE THREAD

**Issue by hassaanelgarem** [wordpress-mobile/WordPress-iOS#18019]
### Expected behavior
Notices should be dismissed in a smooth manner. They should either fade out in place, or fade out while animating downwards out of the screen.

### Actual behavior
Notices glitch while being dismissed.

### Steps to reproduce the behavior
1. Enable quick start for a site
2. Go back to My Site
3. Tap Customize Your Site
4. Tap on a step and follow through
5. When done with a step, a notice for the next step will be automatically shown
6. Press Not now
7. Observe the notice being dismissed with a glitchy animation

https://user-images.githubusercontent.com/25306722/155451991-52f85fa5-aecb-4578-aa78-9bf869d4ae0e.MP4

**This is global to all notices not just quick start notices. Same occurs with notice in grow audience tips from stats**

https://user-images.githubusercontent.com/25306722/155452592-1e8de08d-bad5-46dd-b5ef-378aeb99d395.MP4

**Comment by hassaanelgarem** [comment]
@osullivanchris Can you take a look at this, please? What should be the expected behavior?

**Comment by osullivanchris** [comment]
@hassaanelgarem Hey thank you for raising this, its a good observation.

Historically our use of Snackbars started on the Android app, where they're a [standard out-of-the-box component](https://material.io/components/snackbars). As we've introduced them as a non standard component on iOS I'm assuming we need to define things ourselves.

Here's a quick animation I've created for how I believe it should look/feel

https://user-images.githubusercontent.com/50576199/157233822-68d40192-2b96-4483-b850-0c560aeba4ce.mp4

Properties:
- Snackbars appear from bottom to top. And disappear from top to bottom.
- When they appear/disappear they go from 0->100% opacity (and vice versa) during the duration of the movement.
- I have 'easing' turned on at both the start and end of each transition, so it gradually accelerates then decelerates as it moves (Whether in or out of view)
- I'm moving 60pt up/down from its invisible position
- The duration of the transition is 0.3 seconds
- There is a delay of 3 seconds before the Snackbar disappears automatically

I'm open to suggestions on any of these values. Also depends how you define it in code. But there's a breakdown of how I created this. Happy to discuss/collaborate further.
